pages/docs/tutorials/mpp/multiplatform-library.md

```diff
@@ -267,6 +267,8 @@ class Base64Test {
 
 </div>
 
+Copy gradlew, gradlew.bat and gradle/wrapper/gradle-wrapper.jar from another project setup with gradle.
+
 Execute `./gradlew check` and you will see that the tests are run three times, on JVM, on JS, and on Native!
```

Comment:
gradlew is missing in newly created kotlin multiplatform project using IntelliJ IDEA 2019.3 (Community Edition)

Build #IC-193.5233.102, built on November 27, 2019
Runtime version: 11.0.4+10-b520.11 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.15.1
Non-Bundled Plugins: jd-intellij, org.jetbrains.kotlin